qa/integration/services/elasticsearch.rb

```diff
@@ -0,0 +1,12 @@
+require 'elasticsearch'
+
+class ElasticsearchService < Service
+  def initialize(settings)
+    super("elasticsearch", settings)
+  end
+  
+  def get_client
+    Elasticsearch::Client.new(:hosts => "localhost:9200")
+  end  
+  
+end
```

Comment:
I should have renamed this file `elasticsearch_service` to make sure all the services have the same naming scheme and will be correctly found by the `service_locator`